dotnet/src/Connectors/Connectors.OpenAIV2/Core/Models/PipelineSynchronousPolicy.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Is the folder location correct?

Reply:
That was intentional, but I agree `Models` don't convey a good container for `Policy` specializations, those will be moved as utilities, will do this in a Further PR.